components/movementsensor/gpsutils/cachedData.go

```diff
@@ -15,6 +15,13 @@ import (
 	"go.viam.com/rdk/utils"
 )
 
+// Constants for map keys.
+const (
+	FixValueKey   = 1
```

Comment:
If these keys are just any unique placeholders, [use `iota` instead](https://go.dev/wiki/Iota):
```go
const (
    FixValueKey = iota
    SatsInViewKey
    SatsInUseKey
)
```
edit: Even better would be to get rid of these by using a struct instead of a map; see my other suggestions.